src/hacbs/components/topology/factories/layoutFactory.ts

```diff
@@ -0,0 +1,48 @@
+import { Graph, LayoutFactory, PipelineDagreLayout } from '@patternfly/react-topology';
+import * as dagre from 'dagre';
+import {
+  NODE_SEPARATION_HORIZONTAL,
+  NODE_SEPARATION_VERTICAL,
+} from '../../ApplicationDetails/tabs/overview/visualization/const';
+import { DEFAULT_NODE_SEPERATION_HORIZONTAL } from '../const';
+
+export enum PipelineLayout {
+  WORKFLOW_VISUALIZATION = 'workflow-visualization',
+  PIPELINERUN_VISUALIZATION = 'pipelinerun-visualization',
+}
+
+const DAGRE_SHARED_PROPS: dagre.GraphLabel = {
+  nodesep: NODE_SEPARATION_VERTICAL,
+  ranksep: NODE_SEPARATION_HORIZONTAL,
+  edgesep: 60,
+  ranker: 'longest-path',
+  rankdir: 'LR',
+  marginx: 20,
+  marginy: 20,
```

Comment:
These are the PF default values, so you can drop unless you specifically want to call them out.